care/emr/resources/location/spec.py

```diff
@@ -173,7 +173,7 @@ class FacilityLocationEncounterUpdateSpec(FacilityLocationEncounterBaseSpec):
     status: LocationEncounterAvailabilityStatusChoices
 
     start_datetime: datetime.datetime
-    end_datetime: datetime.datetime | None
```

Comment:
Just to be clear None was already allowed, the FE had to send None in that case, now if the FE does not send anything, BE will assume its None.

Reply:
This case if the value is None, BE might not send anything back at all.

Reply:
Alright, hmm I was getting end_datetime missing message from BE. But no longer an issue since that whole UI was rewritten.